packages/pieces/community/fireflies-ai/src/lib/triggers/new-transcription-complete.ts

```diff
@@ -0,0 +1,155 @@
+import { createTrigger, Property, TriggerStrategy } from '@activepieces/pieces-framework';
+import { firefliesAiAuth } from '../../index';
+import { AuthenticationType, httpClient, HttpMethod } from '@activepieces/pieces-common';
+import { getTranscript } from '../common/queries';
+import { BASE_URL } from '../common';
+
+export const newTranscriptionCompletedTrigger = createTrigger({
+	auth: firefliesAiAuth,
+	name: 'new_transcription_completed',
+	displayName: 'New Transcription Completed',
+	description: 'Triggered when a new meeting is transcribed.',
+	props: {
+		webhookInstructions: Property.MarkDown({
```

Comment:
Fireflies AI supports using a webhook secret key to verify the webhook information but none of that was paid attention to here.

And there is an unnecessary prop defined here `webhookInstructions`. What is the point of it? It does not seem relevant at all to this trigger.